src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+
+const BooksForm = ({ setBooks }) => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setBooks((prevBooks) => [...prevBooks, { title, author }]);
+    setTitle('');
+    setAuthor('');
+  };
```

Comment:
Nice job on this part, its great that you can already submit books!

Reply:
Thanks for the contructive review.